.gitignore

```diff
@@ -4,4 +4,6 @@
 /.vagrant
 .vscode/
 cmd/s2i/debug
+contrib/completions/bash
+contrib/completions/zsh
```

Comment:
why are we ignoring these dirs? Code is still there, so we can't .gitignore without deleting them as well.

Reply:
I've added during running the tests, it's a left over to be removed from this PR. Thanks.